### PR TITLE
Refactor auth to use keycloak.js instead of homegrown system

### PR DESCRIPTION
**Pull request by jacobsee**

### CODE REVIEW

**Comment by jacobsee** [comment]
Must have a change in the private config repository merged at the same time as this one.

File: src/components/drawer_and_header/user_dropdown.tsx [hunk 1]
```diff
@@ -8,10 +8,11 @@ import {
   AnalyticsCategory,
 } from '../../context/analytics_context/analytics_context';
 
-export interface UserDropdownProps {}
+export interface UserDropdownProps { }
 
 export function UserDropdown(props: UserDropdownProps) {
   const authContext = useSession();
+  console.log(authContext)
```
Comment (mcanoy) [inline src/components/drawer_and_header/user_dropdown.tsx]: intended?

Comment (jacobsee) [reply src/components/drawer_and_header/user_dropdown.tsx]: Nope, but a commit from a few hours ago (my latest one) should have removed this... Is this on an older commit of the file?

Comment (mcanoy) [reply src/components/drawer_and_header/user_dropdown.tsx]: stale window maybe?

**Review by mcanoy — APPROVED** [review]
lgtm - About merging. Probably not a big deal for test to be out of sync for a few minutes but if you want precision - I think you have to wait for the container to be built before the MR goes through. I'm not sure if Argo will pick up the new container with MR so 🤷

**Pull request merged**